examples/oft-solana/README.md

```diff
@@ -85,7 +124,7 @@ pnpm hardhat lz:oft:solana:create --eid 40168 --program-id <PROGRAM_ID>
 ```
 
 :important: You may specify the `--additional-minters` flag to add a CSV of additional minter keys to the mint
```

Comment:
Can you fix these :important: tags? We've left them alone for a while, but they deserve to properly flag.

Reply:
will do